gradle.properties

```diff
@@ -11,9 +11,12 @@ springBootCdcVersion=2.1.1.RELEASE
 
 version=0.15.0-SNAPSHOT
 
-eventuateClientVersion=0.20.1.RELEASE
+eventuateClientVersion=0.20.0-SNAPSHOT
 
 debeziumVersion=0.3.6
 dropwizardMetricsVersion=3.1.2
 
 eventuateUtilVersion=0.2.0.RELEASE
+eventuateCdcVersion=0.1.0-SNAPSHOT
```

Comment:
Is this used? See comments about how it should only be used testCompile and for image versions

Reply:
I will fix